index.html

```diff
@@ -0,0 +1,323 @@
+            remote work.
+            <br />
+            I've also been a good helping hand to a lot of developers in
+            different communities!
+            <br />
+            if you'd like to get in contact about potential job opportunities,
+            please reach out!
+          </p>
+          <a
+            href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
+            class="link"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <div class="contact-form">
+        <div class="contact-form-section">
+          <div class="contact-form-description">
+            <h2>Contact me</h2>
+            <p class="contact-me-text">
+              If you have an application you are interested in developing, 
+              a feature that you need to built or a project that you needs coding.
+              I'd love to help with it
+            </p>
+          </div>
+          <div class="contact-form-inputs">
+            <form action="https://formspree.io/f/mqknydla" method="POST">
+              <input type="text" required maxlength="30" name="name" id="txtName" placeholder="Yeremias" />
+              <input type="email" name="email" id="txtEmail" required />
+              <!-- <input 
+                      type="textarea" 
+                      name="details"  
+                      id="txtDetails" /> -->
```

Comment:
- [ ] [REQUIRED] I think it would be better if you remove commented code because it makes code unclean and unprofessional. you should only use comments to explain certain codes. you may use git history if you need to track your previous code.